### DISCUSSION TITLE
Error using Schedule Email API

### DISCUSSION THREAD

**Discussion by andrea-vitolo** [fattureincloud/api#344] [Q&A] [answered]
I always got 422 error when try to POST to /c/{company_id}/issued_documents/{document_id}/email

My call is made through "Make" infrastructure and 

the JSON structure that I'm using is the same provided in api docs:

{
  "data": {
    "sender_email": "mariorossi@fattureincloud.it",
    "recipient_email": "mary.red@example.com",
    "subject": "Nostra pro forma nr. 1",
    "body": "Gentile Mario Rossi,<br>per vedere la nostra pro forma di  o per scaricarne una copia in versione PDF prema sul bottone sottostante.<br><br>{{allegati}}<br><br>Cordiali saluti,<br><b>Mario Rossi</b>",
    "include": {
      "document": false,
      "delivery_note": false,
      "attachment": false,
      "accompanying_invoice": false
    },
    "attach_pdf": true,
    "send_copy": false
  }
}

May you please help me to understand what is wrong?

Many thanks.

Andrea

**Comment by emajo** [comment]
Hi @andrea-vitolo, I think the problem is related to the body field, new lines put like that are breaking the JSON, I suggest to try to change them with `<br/>` tags and see if you still get the error.

I also noticed that you are trying to send the email with the sender `mariorossi@fattureincloud.it`, this will also give an error since you have to put an email for which you configured an SMTP server in Fatture in Cloud or use the default sender as explained [here](https://developers.fattureincloud.it/docs/guides/invoice-creation#-send-the-document).

Thanks,
Emanuele.

**Reply by andrea-vitolo** [reply]
Ciao Emanuele, many thanks for your quick reply.

It's GitHub editor that is changing the json string, following is the json string that I'm using in the body filed when calling the api.

Unfortunately  I got always the same 422 error

<img width="1222" alt="image" src="https://github.com/fattureincloud/api/assets/143037237/d046450f-886d-4c38-b162-65649daa4d23">

**Reply by andrea-vitolo** [reply, marked as answer]
SOLVED, all boolean fields belonging to  "include" are "required" and not "optional"... 

Many thanks Emanuele for your support!

Ciao
